_data/accessibility-tests/tooltip.yml

```diff
@@ -25,6 +25,20 @@ test_items:
     test_type: general
     version_tested: 3.8.0
     wcag_criterion: 1.4.11
+  - summary: You can hover over tooltip content.
+    summary_additional: When you move the pointer over an active tooltip, the tooltip stays open.
+    test_status: fail
+    github_issue_number: 5919
+    github_issue_repo: uswds
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+  - summary: Active tooltips stay open until they are dismissed.
+    summary_additional: Active tooltips close only when focus is moved away from the tooltip trigger or the `escape` key is pressed.
```

Comment:
Recommending a slight revision: Active tooltips close only when you move focus away from the tooltip trigger or press the `escape` key.

Reply:
Thanks! Updated in [0c9c00e](https://github.com/uswds/uswds-site/pull/2646/commits/0c9c00e2f25f7fd9b408d8807ff55992aa5cbd07)